serializer/containerpilot.json5

```diff
@@ -0,0 +1,61 @@
+{
+  consul: 'localhost:8500',
+  jobs: [
+    {
+      name: 'serializer',
+      port: {{.PORT}},
+      exec: 'node /opt/app/',
+      health: {
+        exec: '/usr/bin/curl -o /dev/null --fail -s http://localhost:{{.PORT}}/check-it-out',
+        interval: 2,
+        ttl: 5
+      }
+    },
+    {
+      name: 'consul-agent',
+      exec: ['/usr/local/bin/consul', 'agent',
+                  '-data-dir=/data',
+                  '-config-dir=/config',
+                  '-log-level=err',
+                  '-rejoin',
+                  '-retry-join', '{{ if .CONSUL }}{{ .CONSUL }}{{ else }}consul{{ end }}',
```

Comment:
Just FYI you can you `{{ .CONSUL | default "consul" }}` here. See [TemplateRender / default](https://github.com/joyent/containerpilot/blob/master/docs/30-configuration/32-configuration-file.md#default) in the docs.

I'm not sure how we got in the habit of this particular construct but I know I've repeated it a bunch. But `default` was available when we first implemented templating all the way back in https://github.com/joyent/containerpilot/pull/46! 😀